### PR TITLE
Refactor wording

### PR DESCRIPTION
**Pull request by EagleC0318**
12/1 sprint

### CODE REVIEW

**Comment by tai271828** [comment]
啊我先改 wording 惹，我好像偏向先把你的 refactoring 收下來

**Review by tai271828 — CHANGES_REQUESTED** [review]
This pull requested is oudated because it did not merge `main-tbbca` before editing it. Would you mind resolving the conflicts?

Some example steps for you to resolve the conflicts:
1. Firstly make sure you are on `main-tbbca`
2. `git pull tai main-tbbca` to get the latest `main-tbbca` from tai's repository. `tai` could be any name you `git remote add` pointing to tai's repository
3. `git checkout -b refactor-wording-resolve-conflicts`
4. `git merge main-tbbca` then you should have the conflicts
5. Fix the conflicts
6. Create another pull request, or force-push to override this pull request

Please make sure each of your commits are meaningful. For example, `test` and `revert test` could be removed from your history tree like this one https://github.com/tai271828/disfactory-backend/commit/2f59e5f5d1558341704b2d0bbad230f7bcba30b0 .

**Comment by EagleC0318** [comment]
@tai271828 regarding to this comment https://github.com/tai271828/disfactory-backend/pull/6#pullrequestreview-820503762,
I am wondering why I need to create a new branch for fix the conflict, and how should I remove the non-meaningful commits (like something with `git rebase`)? 
Thanks!

**Comment by tai271828** [comment]
> @tai271828 regarding to this comment [#6 (review)](https://github.com/tai271828/disfactory-backend/pull/6#pullrequestreview-820503762), I am wondering why I need to create a new branch for fix the conflict, 
If you don't want to use a new branch, you may `git push <repo> <branch> --force`

and how should I remove the non-meaningful commits (like something with `git rebase`)? Thanks!
`git rebase -i <some commits>`

File: backend/api/models/const.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class DocumentDisplayStatusConst:
```
Comment (tai271828) [inline backend/api/models/const.py:1]: Append some discussion about the implementation of the namespace https://github.com/tai271828/disfactory-backend/commit/2f59e5f5d1558341704b2d0bbad230f7bcba30b0#commitcomment-61289751

**Comment by tai271828** [comment]
BTW, this change could be upstreamed to Disfactory if you don't mind.

**Comment by tai271828** [comment]
Close this pull request because https://github.com/tai271828/disfactory-backend/pull/8 is landed instead.

**Pull request closed without merging**